src/utils/verify.js

```diff
@@ -0,0 +1,27 @@
+var sodium = require('sodium-universal')
```

Comment:
I extracted the functions we make us of only from the ez lib thing which had many security issues associated. 

This seems to still error out. I need to investigate further

Reply:
Solved in 2ec678975c6f9ca07d77dc4ef4e72db6b8725462